gitlab/assets/dashboards/overview.json

```diff
@@ -0,0 +1,655 @@
+            "time": {},
+            "title": true,
+            "title_align": "center",
+            "title_size": "16",
+            "title_text": "Readiness",
+            "type": "check_status",
+            "width": 14,
+            "x": 18,
+            "y": 11
+        },
+        {
+            "add_timeframe": true,
+            "check": "gitlab.health",
+            "error": null,
+            "group": null,
+            "group_by": [],
+            "grouping": "check",
+            "height": 8,
+            "isShared": false,
+            "sharedGlobalTime": {
+                "live_span": "1h"
+            },
+            "tags": [
+                "*"
+            ],
+            "text_align": "center",
+            "text_size": "auto",
+            "time": {},
+            "title": true,
+            "title_align": "center",
+            "title_size": "16",
+            "title_text": "Health",
+            "type": "check_status",
+            "width": 14,
+            "x": 18,
+            "y": 20
+        },
+        {
+            "add_timeframe": true,
+            "check": "gitlab.prometheus_endpoint_up",
+            "error": null,
+            "group": null,
+            "group_by": [],
+            "grouping": "check",
+            "height": 8,
+            "isShared": false,
+            "sharedGlobalTime": {
+                "live_span": "1h"
+            },
+            "tags": [
+                "*"
+            ],
+            "text_align": "center",
+            "text_size": "auto",
+            "time": {},
+            "title": true,
+            "title_align": "center",
+            "title_size": "16",
+            "title_text": "Endpoint Up",
+            "type": "check_status",
+            "width": 14,
+            "x": 18,
+            "y": 29
+        },
+        {
+            "add_timeframe": true,
+            "auto_refresh": false,
+            "background_color": "gray",
+            "bgcolor": "gray",
+            "content": "Service Checks",
+            "font_size": "18",
+            "height": 25,
+            "html": "Service Checks",
+            "isShared": false,
+            "refresh_every": 30000,
+            "scaleFactor": 1,
+            "show_tick": true,
+            "text_align": "center",
+            "tick": true,
+            "tick_edge": "right",
+            "tick_pos": "50%",
+            "title": false,
+            "type": "note",
+            "width": 15,
+            "x": 1,
+            "y": 12
+        },
+        {
+            "add_timeframe": true,
+            "error": null,
+            "height": 13,
+            "isShared": false,
+            "legend": false,
+            "markers": [],
+            "sharedGlobalTime": {
+                "live_span": "1h"
+            },
+            "tile_def": {
+                "markers": [],
+                "requests": [
+                    {
+                        "q": "avg:gitlab.http_requests_total{*}.as_count()",
```

Comment:
This is the count of new requests, same thing either set the graph type to `bars`, and each bar will be the number of request that happened during the interval.
Or add a `per_second` modifier to graph the number of requests per second